src/App.js

```diff
@@ -1,16 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import chatMessages from './data/messages.json';
+import entries from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import LikeCount from './components/LikeCount';
 
 const App = () => {
+  const [chatData, setChatData] = useState(entries);
+
+  const updateLike = (id) => {
+    const newChatData = chatData.map((chat) => {
+      if (chat.id === id) {
+        return { ...chat, liked: !chat.liked };
+      } else {
+        return chat;
+      }
+    });
+    setChatData(newChatData);
+  };
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat Between Vladimir and Estragon</h1>
```

Comment:
How could we dynamically get the names of the chat participants? What if the chat were just starting and there weren't any messages yet?